go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/go-rod/rod
 go 1.16
 
 require (
+	github.com/icza/mjpeg v0.0.0-20210726201846-5ff75d3c479f // indirect
```

Comment:
But I am not sure how to set the frame rate of a mjpeg file. 

May be I remove the generate avi video function? because the avi file is always very large, I think make use of ffmpeg to record a mp4 file will be better.